extensions/wikia/AdEngine/js/spec/utils/env.spec.js

```diff
@@ -0,0 +1,57 @@
+/*global describe, it, expect, modules, beforeEach, spyOn*/
+describe('ext.wikia.adEngine.utils.hooks', function () {
+
+	function getModule(windowMock, host) {
+		windowMock.location.host = host;
+
+		return modules['ext.wikia.adEngine.utils.env'](windowMock);
+	}
+
+	it('isDevEnvironment returns correct values when run once', function() {
+		var testCases = [
+			{
+				host: 'http://muppet.wikia.com',
+				expected: false
+			}, {
+				host: 'http://muppet.john.wikia-dev.com',
+				expected: true
+			}
+		], windowMock = {
+			location: {
+				host: ''
+			}
+		};
+
+		testCases.forEach(function(testCase) {
+			var module = getModule(windowMock, testCase.host);
+
+			expect(module.isDevEnvironment()).toBe(testCase.expected);
+		});
+	});
+
+	it('isDevEnvironment returns correct values when run more than once', function() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)